### ISSUE TITLE
[ClickHouseResultSet.java] Require null check.

### ISSUE THREAD

**Issue by gmxyb** [housepower/ClickHouse-Native-JDBC#251]
Line 222:
```
@Override
public BigDecimal getBigDecimal(int index) throws SQLException {
  Object data = getObject(index);
  return new BigDecimal(data.toString());  // data may be null
 }
```
Line 238:
```
@Override
 public byte[] getBytes(int index) throws SQLException {
  String data = (String) getObject(index);
  return data.getBytes(cfg.charset());  // data may be null
}
```

**Comment by pan3793** [comment]
Good catch! @gmxyb 

Would you like to send a PR to fix it? Just follow below steps:
- fork repo (I believe you have done it)
- `git clone [your repo url]`
- `git remote add upstream https://github.com/housepower/ClickHouse-Native-JDBC.git`
- `git fetch upstream`
- `git branch --set-upstream-to=upstream/master master`
- Checkout branch from latest master branch `git checkout -b gh-251`
- Then modify the code, don't forget add unit test to verify your code.
- If all things looks great, commit you code with meaningful comment.
- Then push the branch to **your repo**
- Go to Github page, then you will see a tips for send a Pull Request, just follow it!

You can find more details at [Contributing Guide](https://housepower.github.io/ClickHouse-Native-JDBC/dev/contribute.html)

**Comment by pan3793** [comment]
partial fixed by #256
